src/Compilers/Core/Portable/Interop/IClrStrongName.cs

```diff
@@ -132,10 +132,11 @@ internal interface IClrStrongName
             [In, MarshalAs(UnmanagedType.LPWStr)] string pwzFilePath,
             [In, MarshalAs(UnmanagedType.U4)] int dwInFlags);
 
-        [return: MarshalAs(UnmanagedType.I1)]
-        byte StrongNameSignatureVerificationEx(
+        [return: MarshalAs(UnmanagedType.Bool)]
+        bool StrongNameSignatureVerificationEx(
```

Comment:
Would it be better to remove methods that we aren't using?

Reply:
Keeping unused methods is important when dealing with COM signatures. It sets up the vtable ordering.